R/addin.R

```diff
@@ -0,0 +1,33 @@
+#' Add minimal RStudio Addin binding
+#'
+#' This function helps you add a minimal
+#' [RStudio Addin](https://rstudio.github.io/rstudioaddins/) binding to
+#' `inst/rstudio/addins.dcf`.
+#'
+#' @param addin Name of the addin function, which should be defined in the
+#' `R` folder.
+#' @inheritParams use_template
+#'
+#' @export
+use_addin <- function(addin = "new_addin", open = interactive()) {
+  addin_dcf_path <- "inst/rstudio/addins.dcf"
+
+  if (!file_exists(addin_dcf_path)) {
+    create_directory("inst/rstudio")
+    file_create(addin_dcf_path)
+    ui_done("Creating {ui_path(addin_dcf_path)}")
+  }
+
+  addin_info <- render_template("addins.dcf")
```

Comment:
I think you should be able to pass `addin` to render template in order to having to do this by hand. I have a vague recollection that you'll need `{{addin}}` in the template, due to an unfortunate different between glue and whisker.